src/test/java/org/eolang/speco/MainTest.java

```diff
@@ -32,13 +32,17 @@
 import org.apache.commons.io.IOUtils;
 import org.apache.commons.lang3.SystemUtils;
 import org.junit.jupiter.api.Assertions;
-import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.io.TempDir;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
 
 /**
  * Test to test the operation of the command line tool.
  *
  * @since 0.0.1
+ * @todo #22:90min add separate unit-test for each transformation,
+ * which would run step by step and check the intermediate results,
```

Comment:
@advasileva Can we add single indent for `@todo` description?
```
* @todo #22:90min add separate unit-test for each transformation,
*  which would run step by step and check the intermediate results,
*  for example, use yml packs as in dejump.
```
this helps to highlight the text of the task